templates/_helpers.yaml

```diff
@@ -276,3 +276,89 @@ Create the name of the cleanup service account to use
     {{ default "default" .Values.airflow.cleanup.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+
+{{ define "cleanup_task_metadata_airflow_config_path" -}}
+{{ (printf "%s/airflow.cfg" .Values.airflow.airflowHome) | quote }}
+{{- end }}
+
+
+{{ define "cleanup_task_metadata_airflow_metadata_secret" -}}
+{{ default (printf "%s-airflow-metadata" .Release.Name) .Values.airflow.data.metadataSecretName }}
+{{- end }}
+
+{{/* Standard Airflow environment variables */}}
+{{- define "cleanup_task_metadata_standard_airflow_environment" }}
```

Comment:
Why do we need a separate env?

Reply:
I tried to reference as it is - but external template was not accepting those env

Reply:
its fixed now - removed the additional helpers added . Instead called the subchart directly